### DISCUSSION TITLE
VM windows server RDP multiuser

### DISCUSSION THREAD

**Discussion by brair** [AzureCommunityPL/group#470] [Pomoc]
Hej, mam VM z Windows Server 2019, obecnie mogę logować się adminem z 2 komputerów w tym samym czasie, co muszę zrobić aby z 5różnych komputerów można było zalogować się na tą maszynę w tym samym czasie? 
Jeśli potrzebna jest jakaś licencja proszę o informację jaka licencja była by najlepsza device/user? 
Gdzie najlepiej/najtaniej kupić licencję i z niej skorzystać później?
Czy na 1 licencji usera mogę to zrobić? CY muszę jakoś dodatkowo konfigurować server?

Dziękuję z góry za pomoc! :)

**Comment by smereczynski** [comment]
Potrzebujesz skonfigurować RDS i dla każdego usera lub urządzenia potrzebujesz RDS CAL z SA (SAL). SAL można kupić w modelu CSP od partnera i prawdopodobnie to najtańsza opcja. Jeśli nie potrzebujesz Windows Server - to znaczy aplikacja może działać na Windows 10/11, to zainteresuj się Azure Virtual Desktop - będzie taniej i łatwiej.

**Reply by brair** [reply]
Dziękuję za informację!
Czym się różni RDS CAL z SA dla użytkownika a urządzenia? Nie wiem która opcja jest dla mnie najlepsze jeśli jest 5 użytkowników i każdy ma własny PC i chce się łączyć na tą VM z Windows serverem do pulpitu(w tym samym momencie) i indywidualnie uruchamiać w swojej sesji na własnym koncie program?

Sprawdze czy moja aplikacja może działać na Windows 10/11, czy Azure Virtal Desktop zapewni mi taki dostęp jakiego potrzebuje i nie będę potrzebował kupić  licencji dla użytkowników?

**Reply by smereczynski** [reply]
1. Licencja na użytkownika jest przypisana do... użytkownika. Licencja na urządzenie to licencja na konkretny terminal - niezależnie od tego, kto się z niego łączy. Które będzie lepsze? Nie wiem. Jeśli masz X użytkowników, ale łączą się zawsze z tych samych stacji i nigdy równoczesna ilość połączeń nie przekracza ilości stacji, to bierz device. Jeśli User to User i stacja nie ma znaczenia, to bierz User. Jeśli nie wiesz które, to bierz User i obciąż klienta kosztem licencji (life is brutal).
2. W AVD też zapłacisz za licencje, ale zawsze per User i będzie łatwiej.

**Reply by brair** [reply]
Super teraz rozumiem, ale licencję mogę przepinać na innych użytkowników (to na maszynie z Windows serverem przypisuje licencję do użytkownika?) Tzn jeśli usunę konto  użytkownika na serwerze i utworze nowe komto to mogę przypisać tą licencje do nowego konta?
 Czy tak samo jest z device tzn jeśli będę chciał przenieść licencję na nowy komputer? Tak żeby jakikolwiek użytkownik mógł z nowego PC mógł się łączyć do serwera? Tylko wtedy licencję dodaje na komputerze użytkownika? 

Podobno jeszcze jest trial 120dni na RDS ale nie sprawdzałem jak to działa...

**Reply by eklime** [reply]
Dobra, bo widzę, że chyba zabrakło tego istotnego linku: https://learn.microsoft.com/en-us/windows-server/remote/remote-desktop-services/rds-client-access-license

**Reply by brair** [reply]
Cjeśli można to chciałbym zaczerpnąć wiedzy od kogoś kto już to robił zanim N licencji zostanie zakupionych

**Reply by JaroslawBakInitius** [reply]
Ale co dokładnie potrzebujesz wiedzieć?
Dobranie licencji jest jeszcze obwarowane środowiskiem z jakiego korzystasz.
czy to On-Premise? 
Jeśli On-Premise to czy masz klaster Hyper-V czy jest to maszyna fizyczna?
czy może Azure?
jeśli masz Azure lub klaster Hyper-V to aby być zgodnym z licencją powinieneś mieć SA do każdej licencji.
Jeśli to Azure lub inna chmura, to u usługodawcy możesz wynająć licencje SAL tak jak pisał @smereczynski 
Jeśli chodzi o samą instalację to tu wielkich wygibasów nie ma.
Pamiętaj tylko, że ilość licencji per User liczona jest od ilości logujących się użytkowników, a nie od równoczesnego użycia.
Jeśli masz 10 userów logujących się w różnym czasie, to nawet jeśli równocześnie loguje się max 5 to potrzebujesz 10 licencji.

**Comment by JaroslawBakInitius** [comment]
Nie ma w tym wielkiej kombinacji😊
Jak chcesz to zadzwoń

--
z poważaniem
Jarosław Bąk
kom.
+48 501 507 264
